mate-dictionary/src/gdict-applet.c

```diff
@@ -1214,7 +1214,7 @@ gdict_applet_factory (MatePanelApplet *applet,
 }
 
 /* this defines the main () for the applet */
-MATE_PANEL_APPLET_OUT_PROCESS_FACTORY ("DictionaryAppletFactory",
+PANEL_APPLET_FACTORY ("DictionaryAppletFactory",
 			     GDICT_TYPE_APPLET,
 			     "mate-dictionary-applet",
 			     gdict_applet_factory,
```

Comment:
Indents.
```suggestion
PANEL_APPLET_FACTORY ("DictionaryAppletFactory",
		      GDICT_TYPE_APPLET,
		      "mate-dictionary-applet",
		      gdict_applet_factory,
```

Reply:
Indents should now be fixed. This is another mixed tabs and spaces file, and again I got burned on original  8 space tabs vs Pluma defaulting to 4 space tabs. I need to remember to reset that whenever editing probably any file containing tabs in MATE

Reply:
looks good.

Reply:
I can replace tabs with spaces in this file when you are done with it (this and future wayland PRs).

Reply:
Nice, that will make future work much easier